ThirdParty/Kickstart

```diff
@@ -1 +1 @@
-Subproject commit 5e03d0127e514dcf8a63ae9ebd385cd392dcde52
+Subproject commit 290303cd5bedfdc802582a7cb324de1f0619ab08
```

Comment:
https://github.com/OutOfOrder/MonoKickstart is now outdated, https://github.com/MonoGame/MonoKickstart is the one to track.

I agree, there is no need to submodule MonoKickstart.